### PR TITLE
Call out relevant bugs.python.org user info fields

### PR DESCRIPTION
**Pull request by ncoghlan**
- "Login name" is the field to supply as "username" on the CLA
- "Contributor form received" is the field that indicates the CLA
  has been processed

### CODE REVIEW

File: ni/github.py [hunk 1]
```diff
@@ -37,10 +37,11 @@
 For legal reasons we need you to sign this before we can look at your \
 contribution. Please follow these steps to rectify the issue:
 
-1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
-2. **Wait at least one US business day** and then check "Your Details" on \
-[bugs.python.org](https://bugs.python.org/) to see if your account has been \
-marked as having signed the CLA \
+1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/). \
+The "bugs.python.org username" requested by the form is the "Login name" field under \
+"Your Details" on [bugs.python.org](https://bugs.python.org/)
```
Comment (ezio-melotti) [inline ni/github.py]: Perhaps mention that it can be found in the left sidebar.

**Comment by ncoghlan** [comment]
@willingc While updating this PR for @ezio-melotti's comments, I realised it made sense to share more content between the "CLA not signed" case and the "GitHub username missing" case.

For now I've just reused the image you made, but that's missing a highlight arrow for the "Login Name" field.

How did you make the original snippet? I figure it should still be fairly readable even with a 4th arrow added to call out the 3rd relevant field.

**Comment by willingc** [comment]
@ncoghlan I think I made it in Sketch but we should be able to edit it with anything that takes a PNG. I'll update the image (if the URL changes I'll put the new URL here or in a new PR).

**Comment by willingc** [comment]
@ncoghlan Does this work for you? The URL is different and would need to be updated in a few places of this doc file.

image

**Comment by ncoghlan** [comment]
Thanks @willingc! I've updated the references in the PR to use the new version of the screenshot, so I *think* this is ready to go now.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**